src/App.js

```diff
@@ -1,25 +1,53 @@
-import logo from './logo.svg';
-import './App.css';
+import React from "react";
+import Button from "./components/Button";
+import Footer from "./components/Footer";
+//import Footer from "./components/Footer/Footer";
+//import AuctionItem from "./components/AuctionItem";
+import IntroductionAndConclusion from "./components/IntroductionAndConclusion";
+import "./index.css";
 
-function App() {
+const App = () => {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div>
+      <IntroductionAndConclusion
+        title={"Title"}
```

Comment:
When you pass text, you don't need the brackets, so you can do it like this `title="Title"`. You don't need to change this, this is just a heads up